rfcs/text/0000-zendesk-triage-sop.md

```diff
@@ -0,0 +1,150 @@
+be obvious which team should receive the ticket; if so, then it can be
+assigned directly. If it’s not obvious, there can be some internal
+discussion with the secondary support staff about who should receive the
+ticket.
+
+4. At this point the ticket becomes the responsibility of the secondary support person, who should repond to the user and bring the ticket to an eventual close. 
+
+
+In order to improve our User Support the triager will also work with the secondary support team to:
+
+1. tag each ticket with basic tags (data store/ingest/contribution/etc.) that help DCP staff categorize it for later retrieval. In addition, Github tickets should be made for issues like: updating docs or writing new documentation and improving our FAQs, code examples, bug reports, etc. As much as possible, issues should be turned into documentation or short FAQs and made available for users to access from the portal.
+
+2. create and maintain canned responses (how to connect/get involved with the HCA;
+how to submit data, etc.) that can be quickly sent to users who have written with those questions.
+
+3. document discussions and decisions that arise from particular tickets, so that it's clear who should respond to similar questions in the future and what the response should be.
+
+
+## Secondary Support
+
+The secondary support team is comprised of 1-2 members from each major
+component or service of the DCP. These representatives should have
+enough technical knowledge to answer most questions. Answers generated
+by the secondary support team will be added to the knowledge base. They
+may require additional information from the person who created the
+ticket. They also may need to consult with technical experts who make up
+the tertiary support teams.
+
+## Tertiary Support
+
+The Tertiary team is the entire DCP team. In the event that the
+secondary support team is not able to respond to the question, anyone
+may be called on to answer it. However, the need for Tertiary support
+should be rare, and all answers will be archived and added to the
+knowledge
+base
+
+### **Unresolved Questions**
+
+We suggest that funding be sought to hire one or two part or full-time
+triagers who would grow into the position as we move toward GA, and be
+able to handle primary triage as well as some of the more difficult
+questions by the time of the GA, when we expect support needs to
+increase significantly.
+
+> 2 triage agents - could be 50% of their job as they would trade off
+> being point for this job. Duties would include the grooming of the FAQ
+> knowledge base. They would also be responsible for moderate our user
+> forum.
+> 
+> 1 point person per box (4-5 people) - this should be at least 25% of
+> their time, and there should be a backup person for each box
+
+### **Prior Art**
+
+Zendesk is already being used for our support mechanism. This RFC formalizes our processes.
+
+### **Alternatives**
+
+No other alternatives have been proposed.
```

Comment:
One alternative would be to *not* have a support desk unless it's funded. As I noted in Jodi's earlier proposals, this type of frontline-backline support can substantially impact development teams.

Reply:
saying "that the alternative is no support desk, and no one person or team responsible for working through user needs" seems so patently unworkable that I don't want to add it into the RFC.  It is a bit of a challenge in that the RFC can be accepted but not funded, so that won't work either.